test/functional/feature_framework_miniwallet.py

```diff
@@ -31,6 +34,20 @@ def test_tx_padding(self):
                 assert_greater_than_or_equal(tx.get_weight(), target_weight)
                 assert_greater_than_or_equal(target_weight + 3, tx.get_weight())
 
+    def test_wallet_tagging(self):
+        """Verify that tagged wallet instances are able to send funds."""
+        self.log.info(f"Test tagged wallet instances...")
+        node = self.nodes[0]
+        untagged_wallet = self.wallets[0][1]
+        for i in range(10):
```

Comment:
Is the sole reason for creating 10 tagged wallets to be thorough? Or is there any other reason as well?